Strengthen messaging around lack of support for Playground (#6925)
apollographql/apollo-server: docs/source/api/plugin/landing-pages.mdx

</tbody>
</table>

> ## GraphQL Playground landing page

Reviewer: It might be worth leaving a brief section saying that AS3 had a GraphQL Playground landing page and we've done a one-time publish of a new package but otherwise don't support it (ie, no actual example or list of options or anything).

Author: [312add2](https://github.com/apollographql/apollo-server/pull/6925/commits/312add249c878dacd4115cb0670264c4dd15d2a6)

## GraphQL Playground landing page

> By default, Apollo Server 2 provided a GraphQL Playground landing page. For migration purposes, we've published the [`@apollo/plugin-landing-page-graphql-playground`](https://www.npmjs.com/package/@apollo/plugin-landing-page-graphql-playground) package, a GraphQL Playground plugin compatible with Apollo Server 4. However, we aren't supporting this plugin with documentation or security updates. Instead, we recommend migrating to the actively maintained [Apollo Sandbox](/studio/explorer/sandbox) (the default landing page in Apollo Server 4) at your earliest convenience.

Reviewer: Can we keep around (somehow) the mention that GraphQL Playground is officially retired?

Author: @glasser Is it possibly un-retired? This is a fairly recent comment.

https://github.com/graphql/graphql-playground/issues/1366#issuecomment-1062088978

Reviewer: If I interpret this correctly, there's no longer a plan to *merge* the projects. But nobody is actually working on Playground: Rikki opened some issues earlier this year about the idea of continued work but is not planning to do that work personally. And the comment you linked is just saying Playground might be a thing in the future but just a wrapper for GraphiQL? In any case it doesn't seem to be super actively maintained now, and if it ever re-appears it will be quite different from the existing playground that our plugin wraps.  If it comes back to life they can make their own Apollo Server plugin :)

Author: Agreed! Done 👍

> Apollo Server 3 provided a GraphQL Playground landing page by default. For migration purposes, we have published the [`@apollo/plugin-landing-page-graphql-playground`](https://www.npmjs.com/package/@apollo/plugin-landing-page-graphql-playground) package, which is a GraphQL Playground plugin that is compatible with Apollo Server 4. However, we do not intend to support this plugin via documentation, security updates, or otherwise. Instead, we recommend migrating to the actively maintained Apollo Sandbox (the default landing page in Apollo Server 4) at your earliest convenience.

Reviewer 2: I thought AS2 used the GraphQL Playground landing page by default? Did AS3 as well? A few tweaks:
 
```suggestion
By default, Apollo Server 3 provided a GraphQL Playground landing page. For migration purposes, we've published the [`@apollo/plugin-landing-page-graphql-playground`](https://www.npmjs.com/package/@apollo/plugin-landing-page-graphql-playground) package, a GraphQL Playground plugin compatible with Apollo Server 4. However, we aren't supporting this plugin with documentation or security updates. Instead, we recommend migrating to the actively maintained [Apollo Sandbox](/studio/explorer/sandbox) (the default landing page in Apollo Server 4) at your earliest convenience.
```

> Apollo Server 3 provided a GraphQL Playground landing page by default. For migration purposes, we have published the [`@apollo/plugin-landing-page-graphql-playground`](https://www.npmjs.com/package/@apollo/plugin-landing-page-graphql-playground) package, which is a GraphQL Playground plugin that is compatible with Apollo Server 4. However, we do not intend to support this plugin via documentation, security updates, or otherwise. Instead, we recommend migrating to the actively maintained Apollo Sandbox (the default landing page in Apollo Server 4) at your earliest convenience.

Reviewer 2: I thought AS2 used the GraphQL Playground landing page by default? Did AS3 as well? A few tweaks:
 
```suggestion
By default, Apollo Server 3 provided a GraphQL Playground landing page. For migration purposes, we've published the [`@apollo/plugin-landing-page-graphql-playground`](https://www.npmjs.com/package/@apollo/plugin-landing-page-graphql-playground) package, a GraphQL Playground plugin compatible with Apollo Server 4. However, we aren't supporting this plugin with documentation or security updates. Instead, we recommend migrating to the actively maintained [Apollo Sandbox](/studio/explorer/sandbox) (the default landing page in Apollo Server 4) at your earliest convenience.
```

Author: I'm wrong, time flies!
